docs/en/setup/backend/backend-redis-monitoring.md

```diff
@@ -0,0 +1,38 @@
+# Redis monitoring
+## Redis server performance from `redis-exporter`
+SkyWalking leverages redis-exporter for collecting metrics data from Redis. It leverages OpenTelemetry Collector to transfer the metrics to
+[OpenTelemetry receiver](opentelemetry-receiver.md) and into the [Meter System](./../../concepts-and-designs/meter.md).
+
+### Data flow
+1. redis-exporter collect metrics data from Redis.
+2. OpenTelemetry Collector fetches metrics from redis-exporter via Prometheus Receiver and pushes metrics to SkyWalking OAP Server via the OpenCensus gRPC Exporter or OpenTelemetry gRPC exporter.
```

Comment:
Via OpenCensus? I think we only recommend otel.

Reply:
And your codes seem otel already.

Reply:
Yes I just copy other docs. My Fault. I will change them together later.

Reply:
Which docs are still using OC?

Reply:
mysql and posgresql, the ones I submitted before, were not corrected.

Reply:
I re-read it and since we can support opencencus as well and for lower versions of OAP, mentioning opencencus might be necessary?

Reply:
No, OC formats would be for legacy format only. No new code should bundle with it. Google had announced it is merged into otel years ago.
There is no point we waste time on it.

Reply:
Got it, I had modified.